pkg/overlay/config.go

```diff
@@ -46,6 +46,10 @@ type NodeSelectionConfig struct {
 	UptimeCount       int64   `help:"the number of times a node's uptime has been checked" default:"0"`
 	AuditSuccessRatio float64 `help:"a node's ratio of successful audits" default:"0"`
 	AuditCount        int64   `help:"the number of times a node has been audited" default:"0"`
+
+	NewNodeAuditThreshold int64   `help:"the number of audits a node must have to not be considered a New Node" default:"0"`
```

Comment:
shouldn't the default be at least 1? Otherwise a brand new node will not be considered a new node

Reply:
good point!